app/src/main/java/com/movies/presentation/home/custom_view/NavigationView.kt

```diff
@@ -0,0 +1,46 @@
+package com.movies.presentation.home.custom_view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import com.movies.databinding.NavigationCustomViewBinding
+
+class NavigationView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttrs: Int = 0,
+    defStyleRes: Int = 0
+) : FrameLayout(context, attrs, defStyleAttrs, defStyleRes) {
+
+    private val binding =
+        NavigationCustomViewBinding.inflate(LayoutInflater.from(context), this, true)
+
+    init {
+        with(binding) {
+            homeButton.setActiveButton(true)
+            favouritesButton.setActiveButton(false)
```

Comment:
Extract as a function and use it in other places where it is supposed to be  called.

Reply:
done